### DISCUSSION TITLE
Is playtime being tracked?

### DISCUSSION THREAD

**Discussion by haroldm566** [ppy/osu#17089] [Q&A] [answered]
Is playtime already being tracked by lazer, just that it's not visible on profiles atm? Or nah

**Comment by peppy** [comment, marked as answer]
Kind of. It's tracked but not yet processed, so profiles have not been updated with it. You may see that change in the near future, at which point it will *probably* (like 95%) include play time up until now, from the last years.

**Reply by haroldm566** [reply]
Awesome, if it's in the db that's all fine and dandy

**Reply by haroldm566** [reply]
Actually to extend upon this, are other metrics also being tracked internally in lazer as well like most played maps, and maybe medals...?

**Reply by peppy** [reply]
Most played is already tracked. Medals will not be, except medals which are based on statistics, in which case they will still be awarded on next play if you meet the criteria.

**Reply by Sublimelime** [reply]
Is medal-awarding going to be added to lazer soonish or is that a more "final loose ends" kind of thing (after pp and other more important stuff)?

**Reply by peppy** [reply]
Please read [this](https://github.com/ppy/osu-infrastructure/blob/master/score-submission.md).

**Reply by Sublimelime** [reply]
Oh wow I didn't know you straight up did a writeup on this, nice. All questions answered, keep being great <3
